### PR TITLE
[Modular] Newbie pin

### PR DESCRIPTION
**Pull request by desertmote**
## About The Pull Request

It adds a per character preference toggle to spawn with a newbie pin, and is pre-enabled, you lose the pref past 100 living hours.
Its hooked into loadout code because that seemed like the best way to me.

## How This Contributes To The Skyrat Roleplay Experience
Skyrat is a hard server to get into if you're new to spacestation 13, yet people still do. This pin is for players like such, so others are more keen on helping them.

## Proof of Testing

dreamseeker_T2FM0dKpfx
  
</details>

## Changelog
:cl:
add: Players with less than 100 hours as living role get access to a pin indicating they are new.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -159,3 +159,18 @@
...
+	icon_state = "green"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/accessories.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/accessories.dmi'
+
+/obj/item/clothing/accessory/green_pin/examine(mob/user)
+	. = ..()
+	// How many hours of playtime left until the green pin expires
+	var/green_time_remaining = sanitize_integer((PLAYTIME_GREEN - user.client?.get_exp_living(pure_numeric = TRUE) / 60), 0, (PLAYTIME_GREEN / 60))
+	if(green_time_remaining > 0)
+		. += span_nicegreen("It reads '[green_time_remaining] hours.'")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]: Check for it being 1 for the singular "hour"

**Review by tf-4 — CHANGES_REQUESTED** [review]
Cute idea

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]: aka `hour[green_time_remaining >= 2 ? "s" : ""].'`

**Comment by Iamgoofball** [comment]
Did you guys not turn on the Intern feature in the config? This is that but on your ID card.

**Comment by Iamgoofball** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/blob/41b21a967ab08cd07099853289c0b55773091bfa/config/config.txt

Just turn USE_LOW_LIVING_HOUR_INTERN on and set USE_LOW_LIVING_HOUR_INTERN_HOURS to 100 and you're set. The Intern feature even keeps track of who's wearing the ID, so if someone steals it and puts it on the Intern automatically comes off of it, or if an Intern puts someone's ID on it gets Intern appended to it.

**Comment by desertmote** [comment]
> Did you guys not turn on the Intern feature in the config? This is that but on your ID card.

Heyo! only difference I think? is that this pin is a toggle. Is the intern ID a toggle for the player?

**Review by tf-4 — APPROVED** [review]

**Comment by GuiltyNeko** [comment]
> https://github.com/Skyrat-SS13/Skyrat-tg/blob/41b21a967ab08cd07099853289c0b55773091bfa/config/config.txt
> 
> Just turn USE_LOW_LIVING_HOUR_INTERN on and set USE_LOW_LIVING_HOUR_INTERN_HOURS to 100 and you're set. The Intern feature even keeps track of who's wearing the ID, so if someone steals it and puts it on the Intern automatically comes off of it, or if an Intern puts someone's ID on it gets Intern appended to it.

This seems like it would be an undesirable meta-indicator for impersonation.

**Comment by Iamgoofball** [comment]
@GuiltyNeko this is what admins are for

**Pull request merged**